cloudferrylib/os/image/glance_image.py

```diff
@@ -135,7 +135,7 @@ def get_ref_image(self, image_id):
         try:
             return self.glance_client.images.data(image_id)._resp
         except exc.HTTPInternalServerError as e:
-            LOG.error("There is an error occurred while trying to reach "
+            LOG.debug("There is an error occurred while trying to reach "
```

Comment:
Hmmm... this looks like error, let's simply remove this log from here, and log actual message there.